clustered/integration-test/src/test/java/org/ehcache/testing/DynamicConfigStartupBuilder.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.testing;
+
+import org.terracotta.ipceventbus.proc.AnyProcess;
+import org.terracotta.ipceventbus.proc.AnyProcessBuilder;
+import org.terracotta.testing.config.DefaultStartupCommandBuilder;
+import org.terracotta.testing.config.StartupCommandBuilder;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.terracotta.testing.config.ConfigConstants.DEFAULT_CLUSTER_NAME;
+
+public class DynamicConfigStartupBuilder extends DefaultStartupCommandBuilder {
```

Comment:
@saurabhagas :
this class is duplicated from tc-platform and has no link at all with dynamic config code.
Could we move it to galvan ? and remove this from ehcache and tc-platform ?

Reply:
platform should have a galvan-support module like tc-core and tc-ee to support its kit.  This class belongs there.

Reply:
There or in tc-platform in a common module or elsewhere, but the idea is to not have to go over all classes because this is not normal to add a specific behavior whereas dynanmic config becomes the default behavior. Tests in Ehcache should be able to run as-is without any modification.

Reply:
That would also be a place to solve the problem with needing to chose this class in every IT.

Reply:
@saurabhagas : I have opened TDB-4939. I will let you see with @chrisdennis  / @myronkscott about how to solve that. To me, PR works, is usable, can be merged and should be the priority.